spec/src/main/asciidoc/architecture.adoc

```diff
@@ -32,6 +32,39 @@ This specification documents the component specification contents of each releas
 
 The specifications for each release are listed in reverse chronological order.
 
+[[microprofile7.0]]
+=== MicroProfile 7.0
+Based on MicroProfile's time-boxed release process, this is a major release that does include backward incompatible changes. This release no longer packages Jakarta EE 10 Core Profile, instead it just specifies the dependency on Jakarta EE 10 Core Profile. This release does not include MicroProfile Metrics as MicroProfile Telemetry 2.0 offers the Metrics capabilities in addition to Logs and Tracing. Therefore, MicroProfile Metrics became a standalone specification.
+
+If you are still dependent on Jakarta EE 9.1, please consider using the https://github.com/eclipse/microprofile/releases/tag/5.0[5.0 release of MicroProfile].
+If you are still dependent on Jakarta EE 8, please consider using the https://github.com/eclipse/microprofile/releases/tag/4.1[4.1 release of MicroProfile].
```

Comment:
Jakarta EE should not be referenced for spec MP versions that do not "depend on" core profile. There is no direct relationship between MP and Jakarta EE, only individual specs (like JSON-*, REST, CDI)

Reply:
I don't quite understand your above comments. It was the previous version of MP specs that pulled in the Jakarta EE specs. The sentences were copied from the last version of MP spec. I will slightly alter the wording a bit to mention:
If you are still dependent on `the Jakarta EE specifications` from Jakarta EE 9.1...